[Content] Remnant Job To Disable Worldship

rovermicrover opened the pull request:
## Summary
[In another PR adding jobs where the Remnant requested you disable Korath drones was discussed.](https://github.com/endless-sky/endless-sky/pull/7683#issuecomment-1323219564) That idea ended up being scrapped because of up coming content, but I liked the idea of a Remnant job that had you disable instead of kill a foe.

So I added remnant job to disable a 618. I also add an introduction mission for these missions to give them context.

Note the 618 will only spawn in the 3 NW ember waste sectors without a Keystone and a broken jump drive. Meaning even if the player captures the ship they won't be able to get them out of those three sectors.

## Save File
This save file can be used to play through the new mission content:
[Save File](https://www.dropbox.com/s/udre1h6iimx99ok/Rover%20Freewing.txt?dl=0)

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+		"combat rating" > 8103
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request of you."`
```
```suggestion
			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request for you."`
```
alternatively, to ask of you?

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request of you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Ra'gru Ik 618 to be precise. Normally we would request you track down this ship and eliminate, but our scouts also reported that this appears to be home to a large civilian population."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arranged for a prisoner swap with the Korath."`
```
Now, whilst I like the idea of it, I feel like remnant - Korath interactions are based on things that are not said, not explicit arrangements, although they both don't want to harm the other too much, and will try to find the best ways to approach things. So, I'd make it more like this:
```suggestion
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we will find a way to bring them home."`
```
But don't change this until Zitchas gets to look at it

ravenshining replied:
I think this is more in keeping with the way Zitchas envisions these encounters?

```suggestion
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arrange to have the ship disarmed and sent back to a Korath station."`
```

Hurleveur replied:
Thats the thing, how do you bring it back? You fix the JD? Give it another one? That's why I kept it vague

ravenshining replied:
Oh, right, I hadn't thought of *that* aspect.  This will require some thought when I've not stayed up through the night again.

rovermicrover replied:
Added a requirement that you have completed a total of at least 10 Broken Jump drive missions, and changed this that the Remnant will disarm the ship and then fix their jump drive, so the Korath will hopefully decide to just go home.

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arranged for a prisoner swap with the Korath."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have intention of risking my neck for some Korath."`
```
```suggestion
				`	"I have no intentions of risking my neck for some Korath."`
```

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have intention of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Lets make sure everyone gets home safely."`
```
```suggestion
			`	"I am grateful for your understanding and willingness to help here. Let's make sure everyone gets home safely."`
```

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"
+	on enter "Parca"
+		dialog
+			`As you enter the sector your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of the children.`
```
```suggestion
			`As you enter the sector your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of the defenseless Koraths on board, possibly even children.`
```

ravenshining replied:
Korath civilians are... not defenseless >:-)

Also the plural is Korath not Koraths.

```suggestion
			`As you enter the system your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of civilians on board, possibly even children.`
```

Hurleveur replied:
Its a 618, the most military oriented vessel out of all world ships, surely these are not civilians, or they wouldnt have strayed away?

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"The prisoner swap wasn't my idea. I was about ready to fill out the paperwork for the official bounty when one our scouts came to my office to make a personal appeal."`
+			`	"She explained that on her last cloaked fly by scanning run of the 618 she noticed what looked like large group of Korath children playing some type of ball game on the thermal scan."`
+			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 uniq bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`
```
```suggestion
			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 unique bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`
```

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"The prisoner swap wasn't my idea. I was about ready to fill out the paperwork for the official bounty when one our scouts came to my office to make a personal appeal."`
```
Paperwork? idk honestly, but same Zitchas should look at it

ravenshining replied:
Yeah this whole bit is a bit off.  After hundreds of years of fighting Korath, I expect the Remnant know all this by now.  It's also a bit extreme.

rovermicrover replied:
Update with @ravenshining Suggestion resolving this conversation

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
 			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
 			label scanningpayment
 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
+
```
```suggestion



```
3 spaces between the missions

Hurleveur reviewed (commented):
I like this but idk... why a 618? That one is the most heavily armed out of all of them.
We could make this have variants, too

@Zitchas will probably want to take a look at it

ravenshining commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+mission "Remnant: Wont Someone Please Think Of The Children"
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to fight and disable a stranded Ra'gru Ik 618. Do not destroy or capture the ship. Our intel says the world ship has a large civilian population."
```
Wanderers refer to these as world-ships because they don't have much experience with Korath language and culture.  The Remnant should know better.  Also, they don't need intel to know a Korath ship that big is going to be carrying lots of passengers.

```suggestion
	description "Travel to the <waypoints> system to fight and disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
```

ravenshining commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request of you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Ra'gru Ik 618 to be precise. Normally we would request you track down this ship and eliminate, but our scouts also reported that this appears to be home to a large civilian population."`
```
```suggestion
			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Kas'lor Ik 582 to be precise. Normally we would request you track down this ship and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```

ravenshining commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Parca"
+		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"
```
```suggestion
		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"
```

ravenshining commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"The prisoner swap wasn't my idea. I was about ready to fill out the paperwork for the official bounty when one our scouts came to my office to make a personal appeal."`
+			`	"She explained that on her last cloaked fly by scanning run of the 618 she noticed what looked like large group of Korath children playing some type of ball game on the thermal scan."`
+			`	"When she landed she went and looked at the biometric scan data and realized the scan showed around 800 uniq bio signatures, but with only about 7/8th the expected bio mass. Meaning 1/4th of the ships passengers could be children."`
+			`	"I figured you where those children's best chance. I am glad I was not wrong." he says with a content smile on his face.`
+			`	"I worry in our ignorance of Korath society we may have unnecessarily destroyed other ships like this one..."`
+			`	"Expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`
```
```suggestion
			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`
			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these bounties with the proper restraint, expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`
```

rovermicrover replied:
Updated with your suggested text, but couldn't use the commit button because of other changes I had already accepted.

ravenshining commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,26 @@ mission "Remnant: Rescue 5"
+		personality heroic vindictive target marked staying
+		system 
+			near "Parca" 1
+		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"
```
We shouldn't use a constant name for a repeatable job...

```suggestion
		ship "Kas'lor Ik 582 (Crippled)"
```

Hurleveur replied:
```suggestion
		fleet
			names "korath"
			variant
				"Ra'gru Ik 618 (Crippled)"
```
like this then, the way you had done it it would have no name

ravenshining reviewed (changes requested):
I really like this idea, but a 618 is a biiit much.

TODO:
 - Add a crippled variant Kas'lor Ik 582
 - Provide some consequences for failing the mission and the job.

Hurleveur commented:
Shouldnt the ship have some fighters, or maybe an incoming hospital ship to rescue it, in the actual mission?
I don't think this should be written too quickly, and should be fully thought of, including consequences as raven pointed out, but also some context; more details on what that ship did to get there.
Not that I'm against it, of course, but it needs to be fleshed out.

ravenshining commented:
> Shouldnt the ship have some fighters, or maybe an incoming hospital ship to rescue it, in the actual mission? (that you would direct to the ship or smth once its disabled? idk) I

Oh, now that's a good point.  It should really be in a different system, not any of the ones in that Pacra-Levana sector of space, that would be far too frequented by the Korath.  It ought to be trapped somewhere more isolated.

rovermicrover commented:
> > Shouldnt the ship have some fighters, or maybe an incoming hospital ship to rescue it, in the actual mission? (that you would direct to the ship or smth once its disabled? idk) I
> 
> Oh, now that's a good point. It should really be in a different system, not any of the ones in that Pacra-Levana sector of space, that would be far too frequented by the Korath. It ought to be trapped somewhere more isolated.

Somewhere in the graveyard then? Same effect on not being able to capture it and move use it in the greater universe. Though if we used a 577, which the player can already capture with a jump drive, that may not be an issue.

ravenshining commented:
> Somewhere in the graveyard then? Same effect on not being able to capture it and move use it in the greater universe. Though if we used a 577, which the player can already capture with a jump drive, that may not be an issue.

Possibly.  Or closer to Remnant space, either way.

As I said on #7689 , the LKI 577 should *not* be seen outside of Korath space, that it is currently is a mistake.

rovermicrover commented:
Another possibility to explain how they can get the ship out is to add a requirement to the mission that the player's "remnant: broken jump drive count" has to be over something like 10. That way we can add some text to the mission explanations along the lines of "You bringing us broken jump drives has given us enough insight to fix them in many cases, even if we can't create our own."

rovermicrover commented:
> > Somewhere in the graveyard then? Same effect on not being able to capture it and move use it in the greater universe. Though if we used a 577, which the player can already capture with a jump drive, that may not be an issue.
> 
> Possibly. Or closer to Remnant space, either way.
> 
> As I said on #7689 , the LKI 577 should _not_ be seen outside of Korath space, that it is currently is a mistake.

Oh shit I missed that comment in that PR. Will circle back to that PR.

rovermicrover commented:
Added the 582 crippled variant
Added requirement to have completed at least 10 broken jump drive missions.
Added text that the Remnant plan on disarming the ship after it is disabled and then repair the ships jump drive.

rovermicrover commented:
Removed World Ship references with proper ship type
Fixed wrong ship ref in repeat job

rovermicrover commented:
Fixed the spelling mistake and now looking into the outfit issue. A little worried this didn’t show up in my error file… Will try to run this all again locally.

rovermicrover commented:
Ok the outfit issue should be fixed.

Hurleveur commented:
When you change remnant reputation in missions you need to do the same for the "Remnant (Research)" government.
Also remove like 250 reps for failing this, because you gain rep from disabling and killing it and the remnants should really mind it.

rovermicrover commented:
Lost rep changed from -10 to -250 on failure.

Rep change negated if mission aborted.

> Remnant (Research)

I am not seeing this faction rep being changed else where Remnant main rep is changed. Is this still needed.

Hurleveur commented:
Its on abort not on aborted I think
Also you prob dont need to change for both since its only going down.

Could you make += -250 into -= 250 btw?

rovermicrover commented:
on abort fixed

And didn't know -= was supported will change.

rovermicrover commented:
FYI Setup a PR against my own fork so the CI would run if anyone wants to see the CI results
https://github.com/rovermicrover/endless-sky/pull/4

Zitchas commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,37 @@ mission "Remnant: Rescue 5"
+	on complete
+		"remnant: disable and save count" ++
+		payment 2000000
+		dialog "A Remnant military leader thanks you for disabling down the Kas'lor Ik 582 <npc>, and gives you the agreed-upon payment of <payment>."
```
```suggestion
		dialog "A Remnant military prefect thanks you for disabling down the Kas'lor Ik 582 <npc>, and gives you the agreed-upon logistics adjustment of <payment>."
```

While basically being payment by another name, it is good to stick with the "logistics adjustment" framework to maintain consistency. (As in, they aren't *paying* the player, they are providing them with an adjustment to the amount of logistical support they can obtain, as required to deal with damage and expenses incurred in the process of the mission.)

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,58 @@ mission "Remnant: Scent of Rain 1"
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`
```
```suggestion
			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matte. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." he sings with a content smile on his face.`
```

It is subtle, but shifting the concerns from Chilia specifically to the Remnant in general is intentional to suggest and reinforce the idea that there is a significant collaborative decision making process behind the scenes.

Zitchas commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,58 @@ mission "Remnant: Scent of Rain 1"
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`
+			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these bounties with the proper restraint, expect more missions like this in the future." Chilia then walks away invigorated with new purpose.`
```
```suggestion
			`	"We must be careful not to let ignorance of Korath society result in the unnecessary destruction of similar ships. Now that you have shown you can handle these tasks with the proper finesse, you can expect these to turn up in the task board in the future." Chilia then walks away with a confident pace.`
```

Zitchas reviewed (changes requested):
Thanks for writing this! I had several such sorts of missions planned, so this has saved me a bunch of work.

Now, I'll trust Ravenshining to OK the specific ships being used here, but I see she's commented earlier, so that's probably good at this point. 

There are a few technical changes recommended here, but the vast majority are stylistic and writing tweaks to help it match the existing Remnant flavor. All in all, good writing!

rovermicrover commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,58 @@ mission "Remnant: Scent of Rain 1"
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Antevorta"
+		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"
```
Just as context I took three of the name parts from the Korath definition file. So hopefully I understood how the name system works.

ravenshining replied:
Are you sure you were looking at the right file?  Because I can't find bits that would make that up.  If I refer to an older version of the file, however, this can be done:

```suggestion
		ship "Kas'lor Ik 582 (Crippled)" "Fligrookor"
```

rovermicrover commented:
> Thanks for writing this! I had several such sorts of missions planned, so this has saved me a bunch of work.
> 
> Now, I'll trust Ravenshining to OK the specific ships being used here, but I see she's commented earlier, so that's probably good at this point.
> 
> There are a few technical changes recommended here, but the vast majority are stylistic and writing tweaks to help it match the existing Remnant flavor. All in all, good writing!
> 
> edit: Updating your fork & master to the latest from the Endless Sky master may be helpful, too.

Fork updated. All suggestions merged, and running CI on my fork.

Zitchas commented:
Just a side note for future reference: If you look at all the suggestions in the "files changed tab" you should have the option to click on the suggestions and select "add to batch", and then once you have everything selected, click on the "commit batch". Then all the commits are applied together instead of having to do each one individually. :) It's not a problem, it just is easier to follow and a little bit faster.

Zitchas reviewed (approved):
Pending @ravenshining 's approval of the name "Flumgorkor" for the Kas'lor Ik 582, I think this looks good to me.

Nice work!

ravenshining commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
+	on enter "Antevorta"
+		dialog
+			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small city's worth of people, many whose size indicates they are probably children.`
```
It ain't THAT big

```suggestion
			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small town's worth of people, many whose size indicates they are probably children.`
```

ravenshining commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
+	on fail
+		"reputation: Remnant" -= 250
+		dialog
+			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```
```suggestion
			`You were unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

Hurleveur commented on data/korath/korath variants.txt:
```diff
@@ -857,7 +857,30 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor" 5
+		"Microbot Defense Station" 6
+		"Thermal Repeater Rifle" 551
```
You gave rifles to the children?
I warn you, I'll shoot...

Hurleveur replied:
Oh and whilst we're at it add actual children to the ship.
Before outfits, on an added line, add "crew" 500

Hurleveur replied:
Or maybe its best to do it with an invisible outfit that adds required crew actually (invisible as in without category)? I don't have strong opinions on this and since we dont cap it that should be a fine solution.

rovermicrover replied:
The children aren't crew though, and might actually take up less than one bunk. In my head it would make sense to leave them out of all the crew counts. Though yeah, less guns more defense station makes sense either way.

Hurleveur replied:
Sounds good to me, 331 is the default amount of thermal rifles for this ship btw

Hurleveur commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,37 @@ mission "Remnant: Rescue 5"
+		fleet
+			names "korath"
+			variant
+				"Kas'lor Ik 582 (Crippled)"
```
Just thinking, we could maybe have another variant? 
Should prob be done in another PR after this one is merged though. I'm thinking a 577 or a 590?

rovermicrover replied:
582 was selected for these reasons outlined by @ravenshining 
https://github.com/endless-sky/endless-sky/pull/7689#pullrequestreview-1193000780

TLDR Effectively of the civilian ships only 582 should be seen outside the Korath core sectors. And then it should be assumed that it is on some type of refueling mission.

Hurleveur commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,37 @@ mission "Remnant: Rescue 5"
+		"reputation: Remnant" += 250
+	on fail
+		"reputation: Remnant" -= 250
+		dialog
+			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```
I just realized you would get this dialog upon aborting the mission...
```suggestion
		conversation
			branch failed
				not "Remnant: Disable 1: aborted"
			action
				clear "Remnant: Disable 1: aborted"
			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work.`
				decline
			label failed
			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
+		"reputation: Remnant" += 250
+	on fail
+		"reputation: Remnant" -= 250
+		dialog
+			`You were unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```
```suggestion
		conversation
			branch failed
				not "Remnant: Will Not Someone Please Think Of The Children: aborted"
			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
				decline
			label failed
			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

Hurleveur reviewed (commented):
I really like what that mission brings, you dont cap and go out of the remnant systems, you also get payed sufficiently considering you dont get the ship, but can still loot it.
Some things to fix though

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
+	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	minor
+	source
+		government "Remnant"
+		not attributes "station"
```
I don't think an object exists with the Remnant government and the `station` attribute.
Ssil Vida has the `remnant station` attribute, but it seems like the desire is for this mission to be available on Ssil Vida after a certain point.

rovermicrover replied:
Chilia on Ssil Vida seemed weird to me. 

Also I was actually kind of worried about all the sector modification logic that happens with Ssil Vida. So a little bit of fear of the unknown also and reduces possible future issues.

warp-core replied:
The current `to offer` conditions say to me this mission should have the possibility of being offered on Ssil Vida from the end of the Remnant chapter 2 story onwards. If this mission should not be offered there, then the `to offer` conditions should be simplified and this location filter should be changed to `not attributes "remnant station"` since that's the attribute Ssil Vida has.
The attributes of Ssil Vida are not changes at any point in the existing events.
Regardless, `not attributes "station"` is just confusing since its presence doesn't do anything unless I've missed a planet.

rovermicrover replied:
Changed to allow to be offered on Ssil Vida, only source requirement now is Remnant government.

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Antevorta"
```
The conversation says `Parca`, not `Antevorta`.
Also, how is this ship in Antevorta? That's not a trivial journey from the Korath Exile systems, especially not without a keystone of some sort (which the Exiles are either supposed to not have or not know how to use).
Is the idea that this ship had a key stone or similar but lost it?

rovermicrover replied:
It was originally in Parca, but moved it to  Antevorta in response to this comment

https://github.com/endless-sky/endless-sky/pull/7698#issuecomment-1326566610

Fixing the text not being updated in all places though

Hurleveur replied:
Yeah no exile keystones but mb it was the drive malfunctionning?

rovermicrover replied:
Yeah will update the variant to stranded to make it clearer as to what happened.

warp-core replied:
The drive malfunctioning allowing passage through the wormhole is certainly something.

Hurleveur replied:
More like it teleported it far away and then broke

Hurleveur replied:
But could be best to be in Parca still

warp-core replied:
Honestly, it would probably make most sense for it to be trapped in the Postverta cluster.

rovermicrover replied:
IE When you toggle the clusters state via the job "[De]Activate Ssil Vida Power"

rovermicrover replied:
NVM just found the events, it looks like it updates the systems and not replaces them. Can change to that cluster then. Would make more sense.

https://github.com/endless-sky/endless-sky/blob/master/data/remnant/remnant%20events.txt#L344-L1132

Hurleveur replied:
Maybe it should be stranded near Stercutus?

rovermicrover replied:
Changed to Postverta cluster and added Ssil Vida as a spawn point for the side mission.

warp-core commented on data/korath/korath variants.txt:
```diff
@@ -857,7 +857,30 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
 		"Steering (Planetary Class)"
 		"Jump Drive"
 
+ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Crippled)"
+	outfits
+		"Langrage Hyper-Heaver" 3
+		"Warder Anti-Missile" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor" 5
+		"Microbot Defense Station" 6
+		"Thermal Repeater Rifle" 551
+		
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive (Broken)"
```
Generally, the `Crippled` variants of Korath ships are missing more than just their Jump Drive.
Usually, they lack any systems core.
At the stage where this mission is offered, I don't think it's unreasonable to expect the player to be able to take on a world ship with its large systems core, but perhaps this variant should be renamed, perhaps `Stranded` to differentiate it from the usual `Crippled` ships.

Hurleveur reviewed (approved):
The way the mission works sounds good to me now, can't speak for the actual writings and I did add some that may not be the best.
Anyhow, looking forward to it!

Hurleveur commented:
Oh just noticed, it has 3 bays, why are there no fighters? We can make them be in the same system with a different objective btw (aka, none)

Hurleveur commented:
Also you dont need to keep track of the "remnant: disabled and saved" count since that's just how many times the mission was completed

rovermicrover commented:
> Also you dont need to keep track of the "remnant: disabled and saved" count since that's just how many times the mission was completed
> Is it planned for future content?

Was just following the existing pattern from other Remnant missions. This one actually uses the counter from the broken Jump Drive missions so thought it was a cool pattern to keep using.

rovermicrover commented:
> Oh just noticed, it has 3 bays, why are there no fighters? We can make them be in the same system with a different objective btw (aka, none)
> 
> edit: at least for the mission spawned ship we can be sure its the same system, not sure for the job

Can spawn three in on both as NPC evade I think.

Hurleveur commented:
The issue with the job one is you don't know where the ship will be exactly so you can't guarantee they will spawn in the same system? Maybe they do though idk.

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,63 @@ mission "Remnant: Scent of Rain 1"
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessmen ...
```
Capital after the speech ends with a full stop?
Also, did you definitely mean `sings` or was it supposed to be `signs`?

warp-core commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,44 @@ mission "Remnant: Rescue 5"
+				not "Remnant: Disable 1: aborted"
+			action
+				clear "Remnant: Disable 1: aborted"
+			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work.`
```
`remnants`?
Should probably start with a capital and perhaps not have an `s` at the end.

warp-core commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,44 @@ mission "Remnant: Rescue 5"
+			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work.`
+				decline
+			label failed
+			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```
```suggestion
			`You were unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
```

warp-core reviewed (commented):
Is it supposed to be possible to receive the first mission on Esquiline?

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,63 @@ mission "Remnant: Scent of Rain 1"
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	minor
```
Its before

Hurleveur commented:
Can't we just have an empty on abort action?

rovermicrover commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,63 @@ mission "Remnant: Scent of Rain 1"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
+	on enter "Vaticanus"
+		dialog
+			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small town's worth of people, many whose size indicates they are probably children.`
```
Changing this up to remove reference to sensors

warp-core commented:
> Can't we just have an empty on abort action? (like define it and put smth that doesnt matter in it if needed)

An empty action seems pretty undesirable to me.

rovermicrover commented:
> Esquiline

I don't see why not.

rovermicrover commented:
If `on fail` does not trigger on abort if `on abort` is defined then I think the easiest course here would be to have the on abort conversation and nothing else happen on `on abort` and then `on fail` just be the rep loss and failure conversation.

I believe this will result in no rep change on abort, and a rep loss only when the ship is destroyed.

Let me know if the changes look good for this.

Hurleveur commented:
Sounds good to me but do make them dialogs instead of conversations. The conversation was because of the need to have a branching.

rovermicrover commented:
> Sounds good to me but do make them dialogs instead of conversations. The conversation was because of the need to have a branching.

It has been made so

Hurleveur reviewed (approved):
Sounds good to me this way. The dialogs could be in a phrase to have more flavour to them but lets just keep it simple for now, it can be expanded in another PR when we add more variants.
As always, I can't vouch for the writing, only speaking for the mission syntax.

ravenshining commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,36 @@ mission "Remnant: Rescue 5"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Will Not Someone Please Think Of The Children: done"
```
It says below if you abort or fail that it will not be offered again, but it'll offer again anyway.

```suggestion
	to offer
		has "Remnant: Will Not Someone Please Think Of The Children: done"
		not "Remnant: Disable 1: aborted"
		not "Remnant: Disable 1: failed"
```

Hurleveur replied:
I'm still dubious on if we should offer the WS on a plate (its defenceless)and then not offer it again, caus most ppl would just cap it.
My idea when proposing the mission was to encourage looting but I wouldnt mind capping it I guess (also it being in postverta means its locked until we unlock it and that exiles know of it)

Hurleveur replied:
For context, ppl were willing to halt the entire remnant story to get a garanteed WS. Granted this one is cargo and not crew but still.

rovermicrover replied:
The 582 can now be captured and extracted from Korath space. So I think this is an ok trade off.

ravenshining commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
+		random < 5
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
```
```suggestion
		personality heroic target marked staying disables plunders
```

ravenshining reviewed (changes requested):
Looking closer at the variant.  Given the potential for this to be capped, I think this ought to be a bit more difficult.  And, this ship has had a drive failure, it's not been damaged in combat like the crippled variants.

Since the 582 has no fixed guns, we can break the engines instead of the weapons like the crippled variants do without too much detriment to its combat power.  It also makes more sense that the JD blowing would damage the normal engines in the process rather than the weapons.

Additionally, I've added fighters as Hurlever suggested, and tweaked the personality.

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,56 @@ mission "Remnant: Scent of Rain 1"
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
```
why heroic? why not timid?

ravenshining replied:
Consider what they're defending.  A ship with many civilians on board, equipped with long-range artillery?  The `olofezes I think should do whatever it takes.... especially since they can't exactly flee anywhere.

Hurleveur replied:
timid means they stay near the ship and thus are **much** more efficient at defending it

Hurleveur replied:
it seems to work
heroic is the worst thing to give to a fighter it needs to stay near the flagship to get repairs and actually defend it

ravenshining replied:
Cool.  Just mind you this is the mothership not the fighters, I'll change that on them.  They have swarming to keep them near the ship, too.

Hurleveur replied:
I did tell you that I thought swarming ships shoot their own? Anyhow if that's still the case its a separate thing that needs fixing

Hurleveur commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
+			names "korath"
+			variant
+				"Kas'lor Ik 582 (Stranded)"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
```
just checked it doesnt work that way, I wish it did though

Amazinite replied:
This problem of wanting the same random location used by multiple fleets could be solved by #1934.

Hurleveur commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,56 @@ mission "Remnant: Scent of Rain 1"
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Vaticanus"
+		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"
```
best to do this in the ship variant definition if we want it

ravenshining replied:
We could, although I like the idea of making the first mission harder, and having the jobs be more variable.

rovermicrover commented:
Broken Engines I love that solution!

rovermicrover commented:
Realized that it doesn't have a hyper drive so it can't get out anyway. So the issue is already solved lol

ravenshining reviewed (approved):
Looking good

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`
+			`	"Our scouts are reporting a stranded Korath Kas'lor Ik 582 in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```
`a stranded Korath Kas'lor Ik 582 in Vaticanus, a Kas'lor Ik 582 to be precise.`

I think something got messed up in a previous suggestion.

```suggestion
			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```

There should perhaps also be a comma between `operations` and `we`, but I'm not sure. I would put one there but that might just be my writing style. @EjoThims thoughts?

warp-core commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`
+			`	"Our scouts are reporting a stranded Korath Kas'lor Ik 582 in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
+			`	"Would you be up for disabling this Korath Kas'lor Ik 582, so the threat is eliminated while preserving the lives of the innocents on board?" His signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
```
Calm down, Chilia, you don't need to tell me it's a Korath ship and give me the full model name every time, Man is obsessed with saying this.
```suggestion
			`	"Would you be up for disabling it, so the threat is eliminated while preserving the lives of the innocents on board?" His signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
```

warp-core commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
```
`A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space`
This isn't really correct. Postverta and co. aren't bordering on Korath space.
```suggestion
	description "A damaged Kas'lor Ik 582 is stuck in a newly accessible system, near Postverta. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
```

warp-core commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
+		government "Korath"
+		personality heroic target marked staying disables plunders
+		system 
+			near "Vaticanus" 1
```
Just to confirm, this covers Vaticanus, Egeria, and Prosa?
Those systems and those systems only are where you want this to spawn?

rovermicrover replied:
Basically it needs to spawn in that cluster of sectors but not in sector with the station. This was the easiest way I could think of.

warp-core reviewed (commented):
~~Almost done.~~

EjoThims commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`
+			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```
```suggestion
			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations, we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```

EjoThims reviewed (changes requested):
I think it needs the comma, yea.

rovermicrover commented:
@EjoThims is there anything outstanding from you that I haven't responded too?

EjoThims commented:
> @EjoThims is there anything outstanding from you that I haven't responded too?

Oh, I had just weighed in on that comma when pinged. I haven't had a change to look over the thing as a whole yet.

Zitchas commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
+	on visit
+		dialog `You've landed on <planet>, but the stuck Kas'lor Ik 582 has not been disabled yet. Hunt it down and disable it before returning.`
+	on abort
+		dialog `The Remnant are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
```
```suggestion
		dialog `The Remnant are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They advise you they will prioritize other teams for these missions in the future.`
```

Zitchas reviewed (changes requested):
I know that this statement is accurate in terms of this particular mission, but we can't rule out missions similar to this in the future, particularly in the main plot line. 

So this is a minor shift in the tone, and a not so subtle warning that they do, in fact, have other people that can handle these problems.

Zitchas reviewed (approved):
Thanks! I think this looks good.

Amazinite commented on data/korath/korath variants.txt:
```diff
@@ -823,7 +823,33 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
+	turret "Langrage Hyper-Heaver"
+	turret "Banisher Grav-Turret"
+	turret "Banisher Grav-Turret"
+	turret "Warder Anti-Missile"
```
```suggestion
	turret "Warder Anti-Missile"


```

Amazinite commented on data/korath/korath variants.txt:
```diff
@@ -823,7 +823,33 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
 		"Steering (Planetary Class)"
 		"Jump Drive"
 
+ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Stranded)"
```
```suggestion


ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Stranded)"
```

Amazinite commented on data/remnant/remnant jobs.txt:
```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
+			names "korath"
+			variant
+				"Kas'lor Ik 582 (Stranded)"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
```
```suggestion
		dialog `You have disabled the Kas'lor Ik 582. Time to report back to <planet>.`
```

Amazinite commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+		random < 30
+		"remnant: broken jump drive count" >= 10
+		or
+			and
+				has "Remnant: Cognizance 25: done"
+				not "flagship planet: Ssil Vida"
+			has "event: remnant: cognizance calm"
```
```suggestion
			has "event: remnant: cognizance calm"
			and
				has "Remnant: Cognizance 25: done"
				not "flagship planet: Ssil Vida"
```

Amazinite commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+			has "event: remnant: cognizance calm"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`
```
```suggestion
			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you.`
```

Amazinite commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`
+			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations, we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
```
```suggestion
			`	"Our scouts are reporting a stranded Korath ship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations, we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector.`
```

Amazinite commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+		personality heroic target marked staying disables plunders
+		system "Vaticanus"
+		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"
+				crew 551
```
This still doesn't work.
```suggestion
```

Amazinite commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+		system "Vaticanus"
+		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"
+				crew 551
+		dialog `You have disabled the Fligrookor, time to report back to <planet>.`
```
The job just uses the model name, so might as well keep it consistent.
```suggestion
		dialog `You have disabled the Kas'lor Ik 582. Time to report back to <planet>.`
```

Amazinite commented on data/remnant/remnant 2 side missions.txt:
```diff
@@ -715,3 +715,65 @@ mission "Remnant: Scent of Rain 1"
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessmen ...
```
```suggestion
			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." He signs with a content smile on his face.`
```
